widget/embedded/src/containers/ExpandedQuotes/ExpandedQuotes.types.ts

```diff
@@ -0,0 +1,9 @@
+import type { SelectedQuote } from '../../types';
+
+export type PropTypes = {
+  loading: boolean;
+  onClickOnQuote: (quote: SelectedQuote) => void;
+  fetch: (shouldChangeSelectedQuote?: boolean | undefined) => void;
```

Comment:
Do we need `undefined` as a type? the arg is optional.

Reply:
fixed